src/Core/Components/Text/FluentText.razor.cs

```diff
@@ -0,0 +1,138 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// FluentText component, codifies Fluent's opinions on typography to make them easy to use and standardize across products.
+/// </summary>
+public partial class FluentText : FluentComponentBase
+{
+    /// <summary />
+    protected string? ClassValue => new CssBuilder(Class)
+        .Build();
+
+    /// <summary />
+    protected string? StyleValue => new StyleBuilder(Style)
+        .AddStyle("color", GetTextColor(), when: (value) => !string.IsNullOrEmpty(value))
+        .Build();
+
+    /// <summary>
+    /// Gets or sets the tag to be rendered.
+    /// </summary>
+    [Parameter]
+    public TextTag As { get; set; }
+
+    /// <summary>
+    /// Gets or sets the size of the text.
+    /// </summary>
+    [Parameter]
+    public TextSize? Size { get; set; }
+
+    /// <summary>
+    /// Gets or sets the weight of the text.
+    /// </summary>
+    [Parameter]
+    public TextWeight? Weight { get; set; }
+
+    /// <summary>
+    /// Gets or sets the alignment of the text.
+    /// </summary>
+    [Parameter]
+    public TextAlign? Align { get; set; }
+
+    /// <summary>
+    /// Gets or sets the font family of the text.
+    /// </summary>
+    [Parameter]
+    public TextFont? Font { get; set; }
+
+    /// <summary>
+    /// Gets or sets if the texts is set to no-wrap.
+    /// </summary>
+    [Parameter]
+    public bool Nowrap { get; set; }
```

Comment:
Can you avoid the "negation" of properties?
And to use "Wrap" like did by [React](https://react.fluentui.dev/?path=/docs/components-text--docs)
(and with default value to `True`)

Reply:
This is the name of the attribute in the web componnets. Not a fan either but think it is better to keep it the same